client/app/dbaas/logs/detail/offer/logs-offer.controller.js

```diff
@@ -36,13 +36,21 @@ class LogsOfferCtrl {
         this.offerDetail.currentOffer = offerObj.reference;
     }
 
-    saveOffer () {
+    processOrder () {
         this.savingOffer = this.ControllerHelper.request.getArrayLoader({
             loaderFunction: () => this.LogsOrderService.saveOrder(this.serviceName, this.offerDetail)
                 .then(response => this.$window.open(response.order.url, "_self"))
         });
         this.savingOffer.load();
     }
+
+    saveOffer () {
+        if (this.offerDetail.selectedOffer === this.offerDetail.currentOffer) {
```

Comment:
Hmmm.. I'm not sure to understand how this can happen?  Are we sure we want to display a warning message if the user doesn't actually change his offer?  Why not only close the offer edit page?

Reply:
I had actually disabled the order button, but the design changed and currently the behavior they want is to show a warning modal :)

Reply:
as per new design a warning modal need to show if user selects current offer and clicks change button.

Reply:
Ok